lib/CPAN/Uploader.pm

```diff
@@ -68,7 +68,7 @@ sub upload_file {
       . '$file: ' . Data::Dumper::Dumper($file)
     );
   } else {
-    my $tries = ($self->{retries} > 0) ? $self->{retries} + 1 : 1;
+    my $tries = ($self->{retries} || 0 > 0) ? $self->{retries} + 1 : 1;
```

Comment:
Or just `$self->{retries} ? $self->{retries} + 1 : 1;`

Reply:
This seems wrong.  `>` is tighter than `||` so this means:  `$self->{retries} || false`

I'll do something else, though, thanks!